client/scripts/controllers/chain/substrate/account.ts

```diff
@@ -112,12 +120,27 @@ class SubstrateAccounts implements IAccountsModule<SubstrateCoin, SubstrateAccou
         of(api),
         api.derive.staking.validators(),
         api.query.staking.currentEra(),
+        api.derive.staking.electedInfo()
       )),
 
       // fetch balances alongside validators
-      flatMap(([api, { nextElected, validators: currentSet }, era]: [ApiRx, DeriveStakingValidators, EraIndex]) => {
+      flatMap(([api, { nextElected, validators: currentSet }, era, electedInfo]: [ApiRx, DeriveStakingValidators, EraIndex, DeriveStakingElected]) => {
         // set of not yet but future validators
         const toBeElected = nextElected.filter((v) => !currentSet.includes(v));
+        const validatorsInfo: IValidatorInfo = {};
+
+        electedInfo.info.map(({ accountId, exposure: _exposure, validatorPrefs }): any => {
+          const commissionPer = (validatorPrefs.commission) ? (validatorPrefs.commission.unwrap() || new BN(0)).toNumber() / 10_000_000 : 0;
+          const isCommission = !!validatorPrefs.commission;
```

Comment:
have you noticed that the commission doesn't always exist on validators? It seems it always does on Edgeware and Kusama currently.

Reply:
Done.